### DISCUSSION TITLE
Draft feedback and suggestions

### DISCUSSION THREAD

**Discussion by qpetraroia** [Azure/draft#156] [Ideas]
Thanks for joining us at Kubecon NA 2022! This discussion thread will be used to talk about potential features you would like us to automate with Draft. The AKS team is excited about making Draft the best tool to get up and running quickly on Kubernetes, and we look forward to talking with you!

**Comment by sonali-rajput** [comment]
I agree with the documentation part. It will be really helpful for beginners or anyone who wants to know what exactly is happening in those files & manifests. Great session thanks 🤍

**Comment by PixelRobots** [comment]
We need an option to create docker files and yaml that can run the containers as non-route and securely.

**Comment by rod-anami-kyndryl** [comment]
This is very useful and thoughtful tool. I know this depends on volunteers, but it would be nice to see `draft` working with other container registries and kubernetes services.

**Comment by sja-commit** [comment]
Is it possible to use private registry instead of ACR?
